src/appMain/CMakeLists.txt

```diff
@@ -218,18 +218,11 @@ install(
     ${CMAKE_SOURCE_DIR}/mycert.pem ${CMAKE_SOURCE_DIR}/mykey.pem
   DESTINATION bin
 )
-if (REMOTE_CONTROL)
-  install(
-    FILES "${CMAKE_SOURCE_DIR}/src/components/remote_control/remote_sdl_pt.json"
```

Comment:
@LuxoftAKutsan 
If this file is not used anymore (as looks like from #239) I think it makes sense to remove it.